day_7/checker_board.rb

```diff
@@ -0,0 +1,14 @@
+puts "Let's build a checkerboard. You choose how big or small it is!"
+puts "Enter a number"
+
+number = gets.chomp.to_i
+
+number.times do |i|
+ if number > 1
+   puts ("X"*number)[i,number] + "\n"
+ elsif number == 1
+   puts " X X X "
+ else
+   puts "Try Again"
+ end
+end
```

Comment:
This is a good start to building the checkerboard. Remember you need to keep track of building rows all of the same length. Try drawing out a checkerboard one row at a time on a piece of paper and write down the steps you needed for the manual process. Then use that more detailed pseudocode to build your code algorithm.